model-optimizer/extensions/ops/LSTM.py

```diff
@@ -12,11 +12,11 @@ class LSTM(Op):
     def __init__(self, graph: Graph, attrs: dict):
         mandatory_props = {
             'type': 'RNNSequence',  # should be never emitted to IR; for debugging purposes
-            'op': __class__.op,
+            'op': self.op,
```

Comment:
nit: why do you replace `__class__` into `self` everywhere? `__class__` explicitly indicates that this member is static and class related while `self` can confuse that it's instance related.

Reply:
This is our general trend. We do such for all new ops. Also, the IDE complains about the approach with `__class__` and this is a bit annoying.